tests/tests/swfs/avm1/movieclip_begin_gradient_fill/test.toml

```diff
@@ -0,0 +1,8 @@
+ignore = true
+num_frames = 1
+
+[image_comparison]
+tolerance = 0
+
+[player_options]
+with_renderer = { optional = true, sample_count = 1 }
```

Comment:
If there's no output.txt to compare with, make the renderer required as it's only testing the visuals, nothing else. Same for the other test